components/script/dom/bindings/typedarrays.rs

```diff
@@ -176,9 +177,9 @@ pub fn create_new_external_array_buffer(
         NewExternalArrayBuffer(
```

Comment:
Let's add a doc above `free_func`(can't comment on the right line) that explains calling the function must be threadsafe, with a link to https://github.com/servo/mozjs/blob/1d0ba1dc0c40008cfea6f14271844558dbea3cbd/mozjs-sys/mozjs/js/public/ArrayBuffer.h#L89